<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BonusForm</class>
 <widget class="QWidget" name="BonusForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>231</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Factory</string>
   </property>
   <widget class="QRadioButton" name="rbtn_3d_factory">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>3D</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">btnG_factory</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="rbtn_2d_factory">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2D</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">btnG_factory</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_OK">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Shape</string>
   </property>
   <widget class="QRadioButton" name="rbtn_rect_shape">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rectangle</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_circle_shape">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="btnG_factory"/>
  <buttongroup name="btnG_shape"/>
 </buttongroups>
</ui>
